libs/documentation/src/lib/components/formly-search/demos/advanced-search/advanced-search.component.ts

```diff
@@ -0,0 +1,43 @@
+import { Component } from '@angular/core';
+import { FormGroup } from '@angular/forms';
+import { FormlyFormOptions, FormlyFieldConfig } from '@ngx-formly/core';
+
+@Component({
+  selector: 'gsa-sam-advanced-search',
+  templateUrl: './advanced-search.component.html',
+  styleUrls: ['./advanced-search.component.scss']
+})
+export class AdvancedSearch {
+
+  form = new FormGroup({});
+  model: any = {advanceSearch: "Advanced Search"}; 
```

Comment:
Make the model empty for this demo, the link can just have static URL and link text.